nightfall-client/src/routes/commitment.mjs

```diff
@@ -12,6 +12,9 @@ import {
   getWithdrawCommitments,
```

Comment:
A very critical thing I would do, but probably not now, it is to add an `API Key` so that the endpoint wouldn't be freely available to everyone inside the same network, unless some that are supposed to be (e.g. `/health-check`). Please, could you create a ticket for this? (add a configurable API Key for all the nightfall-client endpoints)

Reply:
I think is better you open this ticket. And add what do you think in the description.